docs-src/concepts/what-is-a-payload-codec.md

```diff
@@ -0,0 +1,43 @@
+---
+id: what-is-a-payload-codec
+title: What is a Payload Codec?
+sidebar_label: Payload Codec
+description: A Payload Codec transforms an array of Payloads (for example, a list of Workflow arguments) into another array of Payloads.
+tags:
+  - term
+  - explanation
+---
+
+A Payload Codec transforms an array of Payloads (for example, a list of Workflow arguments) into another array of Payloads.
+
+The Payload Codec is an optional step that happens between the wire and the Payload Converter:
+
+```bash
+Temporal Server <--> Wire <--> Payload Codec <--> Payload Converter <--> User code
```

Comment:
I would reverse the order for simplicity of understanding. User works with User code and I will likely prefer to see it to the left.